tasks

```diff
@@ -209,17 +209,13 @@ function tf_plan {
 
 function tf_plan_db_roles {
   operation=$1
-  db_host=$(_get_aws_ssm_secret "/repo/${NHS_ENVIRONMENT}/output/prm-deductions-ehr-repository/db-host")
-  db_username=$(_get_aws_ssm_secret "/repo/${NHS_ENVIRONMENT}/user-input/ehr-repo-db-username")
-  db_password=$(_get_aws_ssm_secret "/repo/${NHS_ENVIRONMENT}/user-input/ehr-repo-db-password")
-  db_name=$(_get_aws_ssm_secret "/repo/${NHS_ENVIRONMENT}/output/prm-deductions-ehr-repository/db-name")
 
   tf_init_db_roles
```

Comment:
`db_roles`?

Reply:
Relates to the terraform-db-roles folder which also contains IAM policy resources for our DynamoDB table, hence why it has not been removed.